binary/grakn

```diff
@@ -63,10 +63,23 @@ elif [ "$1" = "console" ]; then
     fi
 elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
     if [[ -f "${SERVER_JAR_FILENAME}" ]]; then
-        LIB_CP="server/lib/*"
-        CLASSPATH="${GRAKN_HOME}/${LIB_CP}:${GRAKN_HOME}/server/conf/"
-    # exec replaces current shell process with java so no commands after this one will ever get executed
-        exec java ${GRAKN_DAEMON_JAVAOPTS} -cp "${CLASSPATH}" -Dgrakn.dir="${GRAKN_HOME}" -Dgrakn.conf="${GRAKN_HOME}/${GRAKN_CONFIG}" -Dstorage.javaopts="${STORAGE_JAVAOPTS}" -Dserver.javaopts="${SERVER_JAVAOPTS}" grakn.core.server.GraknServer "${@:2}"
+        while [[ "$#" -gt 0 ]]; do
```

Comment:
if we simplify the while loop to this, will it work?
```
if args[3] == '--debug':
  DEBUG=yes
```

or will it clash with our other flags, such as `grakn server start --benchmark`?

Reply:
It would definitely clash with other flags if we did that - we don't want to enforce that --debug is in a particular position, just that it is there.

Reply:
Right ok